<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMemoryBlockWindow</class>
 <widget class="QMainWindow" name="AddMemoryBlockWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Memory Block</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Button_Confirm">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Back">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>180</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_AddMemoryBlock">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add Memory Block</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="TextField_MemoryBlock">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Tutorial1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Separated by comman (,)</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Tutorial2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Example: 30,20,10</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Tutorial3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Example: 200</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Tutorial4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>50 Memory Size is already in the system.</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
